src/materials/03-organisms/modal-alert-invert.html

```diff
@@ -0,0 +1,32 @@
+<div aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
+  <header class="modal-inner">
+    <h1 class="modal-title t-gamma no-margin">Header</h1>
+  </header>
+  {{> forms.alert-box status="alert invert" text="Email sent. Please check you inbox" classes="no-margin  padding-left--3halves" icon="warning" close-style="text" close-text="Close"}}
+  {{> forms.alert-notice status="alert invert"}}
+
+  <section class="modal-inner">
+    <p class="c-steel">Loreum ipsum</p>
+
+    <div class="form-group">
+      <label>Comment Required</label>
+    {{> forms.textarea }}
+    </div>
+  </section>
+
+  <footer class="modal-footer bg-dust">
+    <div class="modal-button-group row">
+      <div class="modal-button_item">
+        <button class="button {{default l-classes 'alert-fill'}}">{{default left 'Remove'}}</button>
+      </div>
+      <div class="modal-button_item">
+        <button class="button {{default r-classes 'no-border'}}">{{default right 'Cancel'}}</button>
+      </div>
+    </div>
+  </footer>
+
+  <!-- <a class="close-reveal-modal" aria-label="Close"> -->
+  <a class="close-reveal-modal" aria-label="Close"  ng-click="closeModal()">
```

Comment:
@slowbot Do we want to keep `ng-click="closeModal()"` in here, since it's Angular-specific and these modals will be used in both React and Angular?

Reply:
ok thanks sounds good